sdks/python/apache_beam/examples/complete/game/user_score_test.py

```diff
@@ -42,7 +40,6 @@ class UserScoreTest(unittest.TestCase):
       'user1_team1,team1,14,1447697463000,2015-11-16 18:11:03.955',
   ]
 
-  @pytest.mark.examples_postcommit
```

Comment:
I added this mark on a previous PR for the same ticket, but this test https://github.com/apache/beam/blob/aefdc89b927a9e7099fd4a2cc4080babf4ec9790/sdks/python/apache_beam/examples/complete/game/user_score_test.py#L43 was testing only a transform instead of the full pipeline of the example, I created a new test https://github.com/apache/beam/blob/aefdc89b927a9e7099fd4a2cc4080babf4ec9790/sdks/python/apache_beam/examples/complete/game/user_score_it_test.py#L89.